elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/javadsl/ElasticsearchSource.scala

```diff
@@ -168,4 +210,22 @@ object ElasticsearchSource {
     }
   }
 
+  /**
+   * Helper for creating BiFunction instances from Scala
+   * functions as Scala 2.11 does not know about SAMs.
```

Comment:
We'll drop Scala 2.11 for Alpakka 3.0. You can switch it off for Elasticsearch.

Reply:
Added `crossScalaVersions -= Dependencies.Scala211,`